src/source/geojson_source.js

```diff
@@ -185,8 +185,8 @@ class GeoJSONSource extends Evented implements Source {
     /**
      * For clustered sources, fetches the zoom at which the given cluster expands.
      *
-     * @param clusterId The value of the cluster's `cluster_id` property.
-     * @param callback A callback to be called when the zoom value is retrieved (`(error, zoom) => { ... }`).
+     * @param {number} clusterId The value of the cluster's `cluster_id` property.
+     * @param {Callback<number>} callback A callback to be called when the zoom value is retrieved (`(error, zoom) => { ... }`).
```

Comment:
Should this eventually mention the error in the callback? e.g. Callback<Error, number>?